src/sentry/api/event_search.py

```diff
@@ -1103,10 +1113,30 @@ def resolve_field_list(fields, snuba_args, auto_fields=True):
             columns.append("id")
         if not aggregations and "project.id" not in columns:
             columns.append("project.id")
+            project_column = "project_id"
         if aggregations and "latest_event" not in map(lambda a: a[-1], aggregations):
             aggregations.extend(deepcopy(FIELD_ALIASES["latest_event"]["aggregations"]))
         if aggregations and "project.id" not in columns:
             aggregations.append(["argMax", ["project.id", "timestamp"], "projectid"])
+            project_column = "projectid"
```

Comment:
Isn't this `project_id`?

Reply:
You'd think right? 😛, when an aggregation is included we use argMax to find the most recent project id, but the column name then changes to `projectid` instead of `project_id`.